src/libs/ck-libs/ampi/ampiOneSided.C

```diff
@@ -1297,7 +1322,9 @@ int AMPI_Iget_data(void *data, MPI_Status status) {
 AMPI_API_IMPL(int, MPI_Alloc_mem, MPI_Aint size, MPI_Info info, void *baseptr)
 {
   //NOTE: do not use AMPI_API() here, so that the memory allocated is migratable!
-  *(void **)baseptr = malloc(size);
+  *((void **) baseptr) = malloc(size);
```

Comment:
I don't think we need parentheses around `(void **) baseptr`

Reply:
I agree, but doesn't it make the cast clearer?

Reply:
I don't see that

Reply:
removed the parentheses.